ACE/ace/config-integrity-common.h

```diff
@@ -0,0 +1,185 @@
+#ifndef ACE_CONFIG_INTEGRITY_COMMON_H
+#define ACE_CONFIG_INTEGRITY_COMMON_H
+
+#define ghs
```

Comment:
Why use ghs, not a ACE style define

Reply:
This was inheriting from the existing code -- the current code has mixing uses of `ghs` and `INTEGRITY` which seem to have the same meaning. I think it makes sense now to consolidate them to a single macro, may be `ACE_INTEGRITY` as another comment suggests, to follow ACE style.